charts/seed-bootstrap/charts/elastic-kibana-curator/templates/kibana/kibana-configmap.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kibana-configmap
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: kibana-logging
+    role: logging
+data:
+  kibana.yml: |-
+    server.name: kibana
+    server.host: "0"
+
+    elasticsearch.url: "https://elasticsearch-logging:{{ .Values.global.elasticsearchPorts.db }}"
```

Comment:
Is `elasticsearch.url` respected at all? The kibana deployment specifies the same with http:
```
env:
- name: ELASTICSEARCH_URL
  value: http://elasticsearch-logging:{{ .Values.global.elasticsearchPorts.db }}
```